[2025-03-19T11:12:21.108+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-03-19T10:10:00+00:00 [queued]>
[2025-03-19T11:12:21.114+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load scheduled__2025-03-19T10:10:00+00:00 [queued]>
[2025-03-19T11:12:21.114+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-19T11:12:21.123+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2025-03-19 10:10:00+00:00
[2025-03-19T11:12:21.127+0000] {standard_task_runner.py:57} INFO - Started process 1224 to run task
[2025-03-19T11:12:21.129+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'load', 'scheduled__2025-03-19T10:10:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmpaiztglxb']
[2025-03-19T11:12:21.130+0000] {standard_task_runner.py:85} INFO - Job 91: Subtask load
[2025-03-19T11:12:21.142+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-19T11:12:21.167+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_pipeline.load scheduled__2025-03-19T10:10:00+00:00 [running]> on host 9431682cb7a1
[2025-03-19T11:12:21.218+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-19T10:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-19T10:10:00+00:00'
[2025-03-19T11:12:23.633+0000] {logging_mixin.py:154} INFO - Load hoàn tất ❤️
[2025-03-19T11:12:23.696+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-19T11:12:23.703+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load, execution_date=20250319T101000, start_date=20250319T111221, end_date=20250319T111223
[2025-03-19T11:12:23.753+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-19T11:12:23.765+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
